kibana/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 EXPOSE 5601
 
-ENV KIBANA_VER=4.1.2 \
+ENV KIBANA_VER=4.1.11 \
```

Comment:
we also need to update the ES config setting that specifies the kibana version, otherwise we won't be able to accurately set the default index
[1] https://github.com/fabric8io/openshift-elasticsearch-plugin/blob/ES1.5.x/src/main/java/io/fabric8/elasticsearch/plugin/ConfigurationSettings.java#L29